scale/job/models.py

```diff
@@ -391,7 +391,7 @@ def get_details(self, job_id):
                 job.execution = None
         else:
             job.execution = None
-        job.configuration = job.job_type.configuration
+        job.configuration =  convert_config_to_v6_json(job.get_job_configuration()).get_dict()
```

Comment:
Sorry, I wasn't clear earlier because I forgot exactly how I did it. You need to scrub the secrets like we're doing in the job type get_details method.

Reply:
think i got it this time. should be OK as long as we call remove_secret_settings correct?